rsearch/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
```

Comment:
Nothing else has a build.sh, so this is a bit odd.
And because of Godeps, it should probably munge the GOPATH to include the `Godeps/_workspace` path before running `go build`.